src/Processors/Formats/Impl/ParquetBlockInputFormat.cpp

```diff
@@ -54,6 +54,18 @@ namespace ErrorCodes
         }                                                              \
     } while (false)
 
+static Field createDecimalField(const Int256 & value, UInt64 precision_value, UInt64 scale)
+{
+    if (precision_value <= DecimalUtils::max_precision<Decimal32>)
+        return Field(DecimalField<Decimal32>(Decimal32(static_cast<Int32>(value)), static_cast<UInt32>(scale)));
```

Comment:
no need to wrap with Field, keep it simple

Reply:
fix it.